src/usr/local/www/firewall_aliases_edit.php

```diff
@@ -121,6 +121,12 @@ function alias_same_type($name, $type) {
 	$origname = $pconfig['name'];
 }
 
+if ($_REQUEST['exportaliases']) {
```

Comment:
Changes to this file seem to have snuck in from another PR. Remove these and squash the commits (or make a fresh PR)

Reply:
done